css/main.css

```diff
@@ -0,0 +1,393 @@
+}
+
+li {
+  list-style: none;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  text-align: center;
+  color: #fff;
+  padding: 5px;
+}
+
+.card-stacks li {
+  background: rgba(255, 255, 255, 0.24) !important;
+  backdrop-filter: blur(2px);
+  -webkit-backdrop-filter: blur(2px);
+}
+
+.card h2 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.gtime {
+  background-image: url("../media/png/gtime.png");
+}
+
+.weather {
+  background-image: url("../media/png/weather.png");
+}
+
+.conference {
+  background-image: url("../media/png/conference.png");
+}
+
+.katchy {
+  background-image: url("../media/png/katchy.png");
+}
+
+.project-stacks {
+  display: flex;
+  flex-wrap: wrap;
+  grid-gap: 15px;
+  padding: 20px 0;
+}
+
+.project-stacks li {
+  display: inline;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+  background: transparent;
+  color: #fff;
+  transition: 350ms;
+  cursor: pointer;
+}
+
+.project-stacks li:hover {
+  background: rgba(0, 0, 0, 0.349);
+}
+
+.about-me,
+.about-me-section {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.about-me-text {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.link {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  background: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  align-self: start;
+  transition: 350ms;
+}
+
+.link:hover {
+  background: #a04300;
+}
+
+.about-me-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.list-title {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.languages {
+  list-style: inside url("../media/svg/rectangle.svg");
+}
+
+.frameworks {
+  list-style: inside url("../media/svg/square.svg");
+}
+
+.skills {
+  list-style: inside url("../media/svg/circle.svg");
+}
+
+.contact {
+  padding: 0 19% 0 19%;
+  border-top-left-radius: 100px;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  color: #172b4d;
+  text-align: center;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact-input {
+  height: 48px;
+  padding: 15px;
+  margin: 27px 0;
+  border-radius: 20px;
+  border: 1px solid #ced4da;
+  background-color: #fff;
+  min-width: 45rem;
+}
+
+.contact-textarea {
+  resize: none;
+  padding: 2rem;
+  height: 10rem;
+  margin: 27px 0;
+  border-radius: 20px;
+  border: 1px solid #ced4da;
+  color: #212529;
+  background-color: #fff;
+  width: 100%;
+}
```

Comment:
- [ ] Please adhere to the `Figma` design template as much as possible. Currently, your design has some overflow issues and due to that your design is not responsive. So, kindly make sure to fix these issues to create a good `user experience`.
- Please take a look at the below snippets for details.

### Your Design

![Screenshot from 2023-04-20 01-07-26](https://user-images.githubusercontent.com/66476329/233193425-ae150cd0-c9e5-4299-b1b2-82ed6dbb84a8.png)

### Figma Design


![Screenshot from 2023-04-20 01-59-14](https://user-images.githubusercontent.com/66476329/233193538-a22cce0d-93d7-493d-99a1-6f0ed20593da.png)

- [ ] Please also consider adding the `footer` as per project requirements. Adding the footer section which `social` handles help the viewers to connect with you easily on different platforms.